### PR TITLE
1.0: Fix and add links

### PR DESCRIPTION
**Pull request by mishina2228**
* Fix anchors and links.
* Add a link in input/forward.md.

### CODE REVIEW

File: buffer/file.md [hunk 1]
```diff
@@ -90,11 +90,10 @@ This parameter is useful when `.log` is not fit for your environment. See also [
 
 ### Multi-Process Environment
 
-If you use this plugin under the multi-process environment, you need to use `@id`/`root_dir` parameters instead of fixed `path` parameter. See [Multi Process Workers](../deployment/multi-process-workers.md#root_dir-id-parameter) article.
+If you use this plugin under the multi-process environment, you need to use `@id`/`root_dir` parameters instead of fixed `path` parameter. See [Multi Process Workers](../deployment/multi-process-workers.md#root_dirid-parameter) article.
```
Comment (kenhys) [inline buffer/file.md]: It seems no need to modify.

image

File: deployment/signals.md [hunk 2]
```diff
@@ -30,7 +30,6 @@ If you use fluentd v1.9.0 or later, use `SIGUSR2` instead.
 
 ### SIGCONT
 
-Calls SIGDUMP to dump fluentd internal status. See [troubleshooting](trouble-shooting.md#dump-fluentd-internal-information) article.
+Calls SIGDUMP to dump fluentd internal status. See [troubleshooting](trouble-shooting.md#dump-fluentds-internal-information) article.
```
Comment (kenhys) [inline deployment/signals.md:34]: Good catch!

File: input/forward.md [hunk 3]
```diff
@@ -370,7 +370,7 @@ $ echo -e '\x93\xa9debug.tls\xceZr\xbc1\x81\xa3foo\xa3bar' | \
   openssl s_client -connect localhost:24224
 ```
 
-If you can confirm TLS encryption has been set up correctly, please proceed to the configuration of the [`out_forward`](../output/forward.md#how-to-connect-to-a-tls/ssl-enabled-server) server.
+If you can confirm TLS encryption has been set up correctly, please proceed to the configuration of the [`out_forward`](../output/forward.md#how-to-connect-to-a-tlsssl-enabled-server) server.
```
Comment (kenhys) [inline input/forward.md:374]: It should be `../output/forward.md#how-to-connect-to-a-tls-ssl-enabled-server`, right?

image

**Comment by kenhys** [comment]
@mishina2228  Could you check my comments, please?

File: buffer/file.md (hunk 1 above)
Comment (mishina2228) [reply buffer/file.md]: Oh, I was looking at the results on GitHub; it has a different id than GitBook!
rootdirid

File: input/forward.md (hunk 3 above)
Comment (mishina2228) [reply input/forward.md:374]: Same as above, I will fix it!
tslsssl

**Comment by mishina2228** [comment]
Thanks for your prompt review, fixed it!

**Review by kenhys — APPROVED** [review]
LGTM.

**Comment by kenhys** [comment]
@mishina2228 Thanks!

**Pull request merged**